### PR TITLE
5842.7.dev Add chart view to Exchange module

### PR DESCRIPTION
**Pull request by tymoshchuk-o**
https://redmine.devbranch.work/issues/5842

### CODE REVIEW

**Comment by tymoshchuk-o** [comment]
https://redmine.devbranch.work/issues/5842

File: web/modules/custom/green_money_exchange/js/exchange.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+// (
+//   function ($, Drupal, drupalSettings) {
```
Comment (aidua) [inline web/modules/custom/green_money_exchange/js/exchange.js]: commented out code should be removed.

File: web/modules/custom/green_money_exchange/js/exchange.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+//           }
+//         });
+//       }
+//     };
+//   }
+// )(jQuery, Drupal, drupalSettings);
+
+const ctx = document.getElementById('greenChart');
+
+console.log(ctx);
```
Comment (aidua) [inline web/modules/custom/green_money_exchange/js/exchange.js]: development code should be removed.

File: web/modules/custom/green_money_exchange/src/GreenExchangeService.php [hunk 3]
```diff
@@ -165,10 +172,22 @@ public function checkCurrencyStorage(string $date): bool{
 
     $query->condition('exchangedate', $day);
     $data = $query->execute();
-    $cr = $currencyStorage->loadMultiple($data);
+    $currencyList = $currencyStorage->loadMultiple($data);
+
+//    Delete all entity
+//    $currencyStorage->delete($currencyList);
```
Comment (aidua) [inline web/modules/custom/green_money_exchange/src/GreenExchangeService.php]: clean code.

File: web/modules/custom/green_money_exchange/js/exchange.js (hunk 1 above)
Comment (tymoshchuk-o) [reply web/modules/custom/green_money_exchange/js/exchange.js]: In the last commit, all the code was cleaned of unnecessary comments and nine functions

**Pull request merged**